recipes/libmodplug/all/test_package/conanfile.py

```diff
@@ -0,0 +1,21 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self.settings):
+            # Download a public domain xm file; https://modarchive.org/module.php?178293
+            tools.download("https://api.modarchive.org/downloads.php?moduleid=178293#burbs.xm", filename="burbs.xm")
```

Comment:
is it possible copying it to the test package folder instead?

Reply:
I see, that file is 796KB. Let's keep as download.

Reply:
Yes, that's the reason why I have removed it https://github.com/bincrafters/conan-libmodplug/commit/e6c696ce4868ad6675a0f52c087c5071bb8db260

I think we should avoid adding binary blobs to this repository

Reply:
I have enabled the [download cache feature](https://docs.conan.io/en/latest/configuration/download_cache.html) of conan. Not passing the hash to the download function will not cache the download.
Even though it is [not documented](https://docs.conan.io/en/latest/reference/tools.html#tools-download), the `tools.download` had a `sha256=` kwarg argument.

Reply:
Apparently, this got silently added in Conan 1.22.0, since CCI assumes latest Conan version this would be fine with me